docs/administration/configuration/configuring-private-docker-registries/index.md

```diff
@@ -0,0 +1,49 @@
+# Configuring private docker registries
+
+REANA is able to execute workflows which make use of Docker images hosted on private Docker image registries.
+You will just need to save the necessary credentials in Kubernetes secrets, and then instruct REANA to pass these secrets to each job's pod.
+
+## Obtaining the credentials
+
+First of all, you will need to obtain the authentication credentials, and how to do so varies depending on which registry you are using.
+One possibility would be to create a special service account to access the registry.
+As an example, the [REANA account](https://gitlab.cern.ch/reana) is used to authenticate to the private registry provided by CERN's GitLab instance, as shown in [Private Docker registries](../../../advanced-usage/access-control/private-docker-registries/#cern-gitlab).
```

Comment:
Perhaps we can call it the "REANA service account" user, here and in the linked user-facing documentation.